lib/android/app/src/main/java/com/wix/reactnativenotifications/core/notification/PushNotification.java

```diff
@@ -63,11 +63,13 @@ protected PushNotification(Context context, Bundle bundle, AppLifecycleFacade ap
     }
 
     @Override
-    public void onReceived() throws InvalidNotificationException {
-        if (!mAppLifecycleFacade.isAppVisible()) {
+    public void onReceived() {
+        ReactContext reactContext = mAppLifecycleFacade.getRunningReactContext();
+        boolean hasActiveCatalystInstance = reactContext != null && reactContext.hasActiveCatalystInstance();
+        if (!mAppLifecycleFacade.isAppVisible() || !hasActiveCatalystInstance) {
             postNotification(null);
-            notifyReceivedBackgroundToJS();
```

Comment:
Don't we still need to call `notifyReceivedBackgroundToJS()`? Omitting this may be why you cannot start a call from the iOS native contact app (as you mentioned on the status meeting this morning).

Reply:
Does the app still receive `registerNotificationReceivedBackground` events (in `phon-push-notifications/attachEvents.android.js`) without this?

Reply:
idk, I only end-to-end tested and incoming data call worked from dead state.. But I try to check log and and find out..

Reply:
If I comment out the `registerNotificationReceivedBackground` handler, phone still receives incoming data calls, but with no sound 🤷🏽‍♂️

Reply:
Ok, I guess we can live without background notifications. AFAICT we do not consume `push/RECEIVED_BACKGROUND` anywhere. But I don't think this change is suitable for an upstream PR.